### ISSUE TITLE
Pyfa remains locked to the top left of my screen on Mac.

### ISSUE THREAD

**Issue by GKFC** [pyfa-org/Pyfa#2100]
Hi,

I have two screens currently:  My MacBook Pro screen and my monitor.

For some reason my Pyfa window is stuck to the left hand side of my MacBook Pro screen now, I can't move it at all, the regular min / max coloured buttons are gone, and I can only resize the window...

<img width="216" alt="pyfa_20191205" src="https://user-images.githubusercontent.com/24534756/70211919-670c3380-178a-11ea-874b-154cb3c1b094.png">

I've only noticed this happening with the latest version of Pyfa v2.15.1

Cheers,

G.

**Comment by DarkFenX** [comment]
Try following releases:
https://github.com/pyfa-org/Pyfa/releases/tag/v2.10.0
https://github.com/pyfa-org/Pyfa/releases/tag/v2.9.6

Feature implemented in #2015 could be cause of this bug (although I am not confident at all).

**Comment by GKFC** [comment]
I've found a workaround (in case anyone else has an issue):

- Go into System Preferences -> Displays
- Lower the resolution / scaling of the MacBook screen until the navigation bar at the top of Pyfa appears
- Drag the app onto the second monitor
- Change the resolution of the MacBook screen back
- Move the app back onto the MacBook screen and it should now function normally

I THINK it might be more of a Mac-specific bug?  But I'm not sure.

Cheers,

G.

**Comment by GKFC** [comment]
I've found a workaround (in case anyone else has an issue):

    Go into System Preferences -> Displays
    Lower the resolution / scaling of the MacBook screen until the navigation bar at the top of Pyfa appears
    Drag the app onto the second monitor
    Change the resolution of the MacBook screen back
    Move the app back onto the MacBook screen and it should now function normally

If the app is closed you'll have to repeat the above. And clicking on the Pyfa app icon in the dock doesn't function correctly either (i.e. doesn't indicate the app is even open / bring focus to the app when clicked)

I'm not sure what's going on with it tbh.

Cheers,

G.

**Comment by GKFC** [comment]
I've found a workaround (in case anyone else has an issue):

    Go into System Preferences -> Displays
    Lower the resolution / scaling of the MacBook screen until the navigation bar at the top of Pyfa appears
    Drag the app onto the second monitor
    Change the resolution of the MacBook screen back
    Move the app back onto the MacBook screen and it should now function normally

If the app is closed you'll have to repeat the above.

Additionally, it seems to have spawned a new instance or something... there are two apps present in the Toolbar:

![pyfa_toolbar_20191209](https://user-images.githubusercontent.com/24534756/70397399-facd5080-1a65-11ea-99d0-ec27f8f58d37.png)

Only one of them actually affects the app itself when clicked though.

I'm not sure what's going on with it tbh.

Cheers,

G.

**Comment by DarkFenX** [comment]
Do you have to do it every restart? Have you tried the releases i linked?

**Comment by GKFC** [comment]
Yes I have to do that every restart to "fix" it.  And then the same bugs reappear.

Hadn't looked at the previous releases, only noticed it recently when I moved the window.

v2.9.6 seems fine (although data is old obviously), v2.10.0 still has the problem.

Cheers,

G.

**Comment by DarkFenX** [comment]
Ok so it seems that the code I added to restore window position breaks it. I will try to look into it before next release.

**Comment by GKFC** [comment]
Hi,

Just checking in to see if there was an ETA for this one?  It's still happening and means I can't really move my Pyfa around the screen easily.

Cheers,

G.
